templates/pkg/resource/sdk_find_read_one.go.tpl

```diff
@@ -34,6 +34,9 @@ func (rm *resourceManager) sdkFind(
 {{- end }}
 	rm.metrics.RecordAPICall("READ_ONE", "{{ .CRD.Ops.ReadOne.ExportedName }}", err)
 	if err != nil {
+		if reqErr, ok := ackerr.AWSRequestFailure(err); ok && reqErr.StatusCode() == 404 {
+			return nil, ackerr.NotFound
++       }
```

Comment:
+ :eyes:

Reply:
@embano1 I think what @A-Hilaly is trying to say here is that the indentation is off here :) `gofmt` cleans it up in the generated code but for our eyes in reviewing PRs that touch the templates, it's best to replace four spaces with `\t` characters in the files before saving.

In Vim I do: `:%s/    /\t/gI`

Reply:
i meant \+ :eye: , there is a tiny extra character in the last line :)

Reply:
Intentation is correct, but there was a little `+` copied from my `git patch` foo, well spotted @A-Hilaly !